README.md

```diff
@@ -7,7 +7,7 @@
     go get github.com/ceph/go-ceph
 
 The native RADOS library and development headers are expected to be installed.
-
+(librados-devel and librbd-devel)
```

Comment:
i can remove this line if you wish .. ( i found it usefull to have the pkgs but maybe in a debian context they differ) as you wish :smile_cat:

Reply:
This is really helpful. In the README.md could you make a sub-section with dependencies and note that these two are for RHEL/CentOS/Fedora? Also, submitting the documentation changes under a different PR than the `guardclause for functions` would be nice. Other than that, it all looks good and we can get everything merged. Thanks!